print-engine/src/main/java/de/tillhub/printengine/data/ExternalPrinter.kt

```diff
@@ -0,0 +1,7 @@
+package de.tillhub.printengine.data
+
+class ExternalPrinter(
+    val info: PrinterInfo,
+    val interfaceType: String,
+    val manufacturer: ExternalPrinterManufacturer
```

Comment:
Delete ExternalPrinterManufacturer and change to String with comment explanation what can it be.